README.md

```diff
@@ -227,57 +227,99 @@ _Default to `null`._
 
 ## Usage in Symfony
 
-### Client as a service
+### Configuration
 
-Just declare the proper service in `services.yaml`:
+You'll need to add the bundle in `bundles.php`:
+
+```php
+// config/bundles.php
+return [
+    // ...
+    JoliCode\Elastically\Bridge\Symfony\ElasticallyBundle::class => ['all' => true],
+];
+```
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏Could it be automatic with Symfony Flex? 🤔 Maybe not in the context of a bridge?

Reply:
Yes! I plan to explain everything here, and then submit a PR to the recipes repo, and then update this repo 👍🏼